Lib/calendar.py

```diff
@@ -157,6 +157,35 @@ def weekday(year, month, day):
         year = 2000 + year % 400
     return Day(datetime.date(year, month, day).weekday())
 
+#for a user-defined additional holiday list, you can pass a list to the argument
+def get_weekends_count(start_date , end_date):
+    #find the number of weekends between start_date and end_date
+    #NOTE:::: START DATE < END DATE ALWAYS
+            start_date_weekday = start_date.weekday()
+            end_date_weekday = end_date.weekday()
+            first_day_of_week_dayoftheweek = start_date_weekday
+            total_days = (end_date - start_date).days + 1 #caution
+
+            div = int(total_days / 7)
+            rem = int(total_days % 7)
+
+            num_weekends = div * 2
+            
```

Comment:
Remove trailing space:

```suggestion

```

Reply:
@NobitaNoobie, you haven't removed the trailing space. It causes the lint check to fail.
Your code should also be formatted according to https://peps.python.org/pep-0008.

Reply:
Note that you can make edits to https://github.com/NobitaNoobie/cpython/tree/patch-3 and the pull request will be updated accordingly. You can commit directly on that branch.